fileblock/src/inode.rs

```diff
@@ -11,7 +11,7 @@ enum FileType {
 struct Inode {
     inumber: u32,
     ftype: FileType,
-    fsize: u64,
-    block_location_id: u32,
+    fsize: u32,
```

Comment:
i'm back to thinking this should be bytes as in in the exact number of byes in my file object. 

wdyt @allancalix ?